docs/tutorials/quick-start/index.rst

```diff
@@ -0,0 +1,645 @@
+    issuer.certmanager.k8s.io "letsencrypt-prod" created
+
+Both of these issuers are configured to use the :doc:`HTTP01 </reference/issuers/acme/http01>` 
+challenge provider.
+
+Check on the status of the issuer after you create it:
+
+.. code-block:: shell
+  :emphasize-lines: 28-32
+
+    $ kubectl describe issuer letsencrypt-staging
+
+    Name:         letsencrypt-staging
+    Namespace:    default
+    Labels:       <none>
+    Annotations:  kubectl.kubernetes.io/last-applied-configuration={"apiVersion":"certmanager.k8s.io/v1alpha1","kind":"Issuer","metadata":{"annotations":{},"name":"letsencrypt-staging","namespace":"default"},"spec":{"a...
+    API Version:  certmanager.k8s.io/v1alpha1
+    Kind:         Issuer
+    Metadata:
+      Cluster Name:
+      Creation Timestamp:  2018-11-17T18:03:54Z
+      Generation:          0
+      Resource Version:    9092
+      Self Link:           /apis/certmanager.k8s.io/v1alpha1/namespaces/default/issuers/letsencrypt-staging
+      UID:                 25b7ae77-ea93-11e8-82f8-42010a8a00b5
+    Spec:
+      Acme:
+        Email:  your.email@your-domain.com
+        Http 01:
+        Private Key Secret Ref:
+          Key:
+          Name:  letsencrypt-staging
+        Server:  https://acme-staging-v02.api.letsencrypt.org/directory
+    Status:
+      Acme:
+        Uri:  https://acme-staging-v02.api.letsencrypt.org/acme/acct/7374163
+      Conditions:
+        Last Transition Time:  2018-11-17T18:04:00Z
+        Message:               The ACME account was registered with the ACME server
+        Reason:                ACMEAccountRegistered
+        Status:                True
+        Type:                  Ready
+    Events:                    <none>
+
+You should see the issuer listed with a registered account. 
+
+Step 5 - Deploy a TLS Ingress Resource for the Echoserver
+=========================================================
+
+With all the pre-requisite configuration in place, we can now do the pieces to request 
+the TLS certificate. There are two primary ways to do this: using annotations on the ingress
+with :doc:`ingress-shim </reference/ingress-shim>` or directly creating a certificate resource. 
+
+In this example, we will add annotations to the ingress, and take advantage of ingress-shim to 
+have it create the certificate resource on our behalf. After creating a certificate,
+the cert-manager will update or create a ingress resource and use that to validate the domain.
+Once verified and issued, cert-manager will create or update the secret defined in the certificate.
+
+.. note::
+
+    The secret that is used in the ingress should match the secret defined in the certificate.
+    There isn't any explicit checking, so a typo will resut in the nginx-ingress-controller
+    falling back to its self-signed certificate.
+
+Edit the ingress add the annotations that were commented out in our earlier example:
+
+.. literalinclude:: example/ingress-tls.yaml
+   :language: yaml
```

Comment:
Maybe an emphasis lines here?

Reply:
hmm, good question - not sure if `emphasize lines` works with `.. literalinclude`, but I'll find out!

Reply:
yep! Added!